src/util_c_unittest.nss

```diff
@@ -73,15 +73,19 @@ const int UNITTEST_ALWAYS_EXPAND = FALSE;
 
 /// @brief Custom handler to handle reporting unit test results.
 /// @param sOutput The formatted and colored output results of a unit test.
-void HandleUnitTestOutput(string sOutput)
+void HandleUnitTestOutput(string sOutput, object oTarget=OBJECT_INVALID)
```

Comment:
This function is called from `util_i_unittest` from within functions that don't receive a target object, so including `oTarget` would be useless without also modifying which target is sent from `util_i_unittest`.  Since it's not called directly by the user creating unit tests, the `oTarget` parameter would have to be integrated into the various test calls to be effective.